### PR TITLE
Katello 3.7 RC1

### PR DESCRIPTION
**Pull request by jturel**
For plugin updates, please indicate which repos this should be built into:

* [] Nightly
* [x] 1.18
* [ ] 1.17
* [ ] 1.16

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ehelms** [comment]
Also need to clear out -- https://github.com/theforeman/foreman-packaging/blob/rpm/1.18/package_manifest.yaml#L292

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -4,7 +4,7 @@
 %global foreman_min_version 1.18.0
 %global plugin_name katello
 %global gem_name katello
-# %%global prever .rc1
+%global prever .rc1
```
Comment (ehelms) [inline packages/katello/rubygem-katello/rubygem-katello.spec]: Needs changelog

File: packages/katello/katello-installer/katello-installer-base.spec [hunk 2]
```diff
@@ -2,11 +2,11 @@
 %undefine scl_prefix
 %global scl_ruby /usr/bin/ruby
 
-# %%global prever .rc1
+%global prever .rc1
 
 Name:    katello-installer-base
 Version: 3.7.0
-Release: 1.nightly%{?prever}%{?dist}
+Release: 1%{?prever}%{?dist}
```
Comment (ehelms) [inline packages/katello/katello-installer/katello-installer-base.spec]: Needs changelog

**Comment by jturel** [comment]
[test rpm]

**Comment by ehelms** [comment]
[test rpm]

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 3]
```diff
@@ -4,15 +4,15 @@
...
 %global mainver 3.7.0
-%global release 1.nightly
+%global release 1
 
 Name:    %{?scl_prefix}rubygem-%{gem_name}
 Summary: Content and Subscription Management plugin for Foreman
 
 Version: %{mainver}
-Release: %{?prever:0.}%{release}%{?prever}%{?dist}
+Release: %{release}%{?prever}%{?dist}
```
Comment (ehelms) [inline packages/katello/rubygem-katello/rubygem-katello.spec]: This line doesn't actually need to be changed

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 4]
```diff
@@ -183,7 +183,7 @@ Documentation for %{pkg_name}.
 gem unpack %{SOURCE0}
 %{?scl:EOF}
 
-%setup -q -D -T -n  %{gem_name}-%{version}
+%setup -q -D -T -n  %{gem_name}-%{version}%{?prever}
```
Comment (ehelms) [inline packages/katello/rubygem-katello/rubygem-katello.spec]: Sorry, this isn't actually needed, I was looking at this all wrong. Basically, let's mimic this https://github.com/theforeman/foreman-packaging/commit/5f45c183be9b087ce87f24bc4b87d86120d587b8#diff-a8930fa5b28236b4e1d2d7dc0775f97b

**Comment by ehelms** [comment]
Need to drop fedora25 and fedora26 from https://github.com/theforeman/foreman-packaging/blob/rpm/1.18/rel-eng/releasers.conf#L31

I musta missed that.

**Comment by jturel** [comment]
[test rpm]

**Pull request merged**